### PR TITLE
Update paradox

### PR DESCRIPTION
**Pull request by raboof**
Which adds checks for dead references and pages that are unexpectedly not
in any table of contents

### CODE REVIEW

**Comment by helena** [comment]
Nice!

File: project/plugins.sbt [hunk 1]
```diff
@@ -17,7 +17,7 @@ addSbtPlugin("io.spray" % "sbt-boilerplate" % "0.6.1")
 addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.4")
 addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.18")
 addSbtPlugin("com.lightbend.paradox" % "sbt-paradox-apidoc" % "0.2")
-addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.5.4")
+addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.6.1")
```
Comment (helena) [inline project/plugins.sbt]: Will be nice to have the upgrades :)

**Review by helena — APPROVED** [review]
LGTM

**Review by ennru — APPROVED** [review]
LGTM.

**Pull request merged**